src/java.datatransfer/share/classes/java/awt/datatransfer/DataFlavor.java

```diff
@@ -1158,7 +1157,7 @@ public final String getDefaultRepresentationClassAsString() {
      *         {@code java.io.InputStream}
      */
     public boolean isRepresentationClassInputStream() {
-        return ioInputStreamClass.isAssignableFrom(representationClass);
+        return java.io.InputStream.class.isAssignableFrom(representationClass);
```

Comment:
```suggestion
        return InputStream.class.isAssignableFrom(representationClass);
```
The class is already imported on line 32.

Reply:
This is the style used by the various methods in this class to highlight an exact used type.